src/util.coffee

```diff
@@ -32,3 +32,25 @@
                                       node?.tag is 'tag:yaml.org,2002:map'
 
 @NON_PRINTABLE = /[^\x09\x0A\x0D\x20-\x7E\x85\xA0-\uD7FF\uE000-\uFFFD]/
+
+@stringifyJSONCircularDepth = (json, depth = 1000) ->
```

Comment:
Hey @repocho, you are absolutely right, this is the only way to handle recursive data structures. 1000 seems a little high still.

If you have signed this: https://api-notebook.anypoint.mulesoft.com/notebooks#bc1cf75a0284268407e4

I'll merge and publish. Thanks for your contrib.